spring-cloud-gcp-data-datastore/src/main/java/org/springframework/cloud/gcp/data/datastore/core/DatastoreTemplate.java

```diff
@@ -786,6 +790,10 @@ private void maybeEmitEvent(ApplicationEvent event) {
 		}
 	}
 
+	void setMaxWriteSize(int maxWriteSize) {
```

Comment:
Does this need to be configurable? I.e. think it should be fixed to 500 constant since that is the max.

Reply:
we need that for tests